server/lib/pdf.js

```diff
@@ -0,0 +1,24 @@
+import config from 'config';
+import fetch from 'node-fetch';
+
+import { TOKEN_EXPIRATION_LOGIN } from '../lib/auth';
+
+export const getTransactionPdf = async (transaction, user) => {
+  if (['ci', 'test'].includes(config.env)) {
+    return;
+  }
+  const pdfUrl = `${config.host.invoices}/transactions/${transaction.uuid}/invoice.pdf`;
+  const accessToken = user.jwt({}, TOKEN_EXPIRATION_LOGIN);
+  const headers = {
+    Authorization: `Bearer ${accessToken}`,
+  };
+  return await fetch(pdfUrl, { method: 'get', headers }).then(response => {
```

Comment:
If the remote server doesn't reply, `fetch` will throw an error (try `fetch('https://xxxxxjuk846yk78y4k6i84k354etw5t4xx.com/qqqqq').then(resp => console.log(resp))`). 

We should properly handle that to make sure we still send the thank you email, even if there's to PDF attached.